src/python/pants/engine/legacy/structs.py

```diff
@@ -60,7 +60,7 @@ def get_sources(self) -> Optional["GlobsWithConjunction"]:
     if sources is None:
       if self.default_sources_globs is None:
         return None
-      default_globs = Globs(
+      default_globs = Files(
```

Comment:
Why can we do this?

Reply:
`Files` and `Globs` behave identically here as `Files.legacy_globs_class == wrapped_globs.Globs == Globs.legacy_globs_class`. I did this as minor prework for soon removing `structs.Globs`.

Reply:
Ok, awesome! Sounds great!